### PR TITLE
NOJIRA: adding more precise instructions

### PR DESCRIPTION
**Pull request by vmindru**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,9 +28,14 @@ instance of the CLI:
 The integration tests do this automatically, so this step can be skipped if
 those have already been run.
 
+GCC is needed to compile vagrant plugins 
+
+    $ yum install qemu libvirt libvirt-devel ruby-devel gcc qemu-kvm
```
Comment (shaded-enmity) [inline README.md]: There's a `yum install` command below this one as well, can you merge them?

Comment (vmindru) [inline README.md]: done

File: README.md [hunk 2]
```diff
@@ -28,13 +28,14 @@ instance of the CLI:
 The integration tests do this automatically, so this step can be skipped if
 those have already been run.
 
-The prototype requires Vagrant with some relevant plugins:
+GCC is needed to compile vagrant plugins 
+
+    $ yum install qemu libguestfs-tools-c libvirt libvirt-devel ruby-devel gcc qemu-kvm
```
Comment (shaded-enmity) [inline README.md]: we should `sudo` here as well for consistency

**Comment by shaded-enmity** [comment]
one minor nitpick, other than that LGTM

File: README.md (hunk 2 above)
Comment (vmindru) [inline README.md]: fixed

**Comment by shaded-enmity** [comment]
thanks, merging!

**Pull request merged**